### ISSUE TITLE
Clear chat is not being announced

### ISSUE THREAD

**Issue by reginacas** [reginacas/sense-ui#14]
## Description
For one tester, the "clear chat" command did not announce the action. 

## Origin

As noticed by @paulGeoghegan : 

> It does clear the chat but I'm not told anything. You should probably get a message about it or something.

This problem was also noticed by @samughal819 and @qadeer-wadera
